linux/README.MD

```diff
@@ -0,0 +1,29 @@
+Reverse Suite for Linux
+=============
+
+```bash
+# ./rsuite.sh -h
+  _____                                 _____       _ _       
+ |  __ \                               / ____|     (_) |      
+ | |__) |_____   _____ _ __ ___  ___  | (___  _   _ _| |_ ___ 
+ |  _  // _ \ \ / / _ \ '__/ __|/ _ \  \___ \| | | | | __/ _ \ 
+ | | \ \  __/\ V /  __/ |  \__ \  __/  ____) | |_| | | ||  __/
+ |_|  \_\___| \_/ \___|_|  |___/\___| |_____/ \__,_|_|\__\___|
+                                                                                                                            
+./rsuite.sh <options>, where <options> are:
+
+     -h                        what you are looking at
+     -d <file.apk>,            disassemble the apk (convert to smali)
+     -l,                       shows the name of the applications already disassembled.
+     -b <application_name>,    build the application as an apk 
```

Comment:
Maintain coherency between these lines and their dots (.)

Some lines have dots and others don't